README.rst

```diff
@@ -39,6 +38,48 @@ a virtual environment):
     $ pip install -e .
 
 
+Building a Python ZIP using `shiv <http://shiv.readthedocs.io/en/latest/>`_
+---------------------------------------------------------------------------
+
+To create a .pyz with all dependencies included, you'll need to `install shiv <https://github.com/linkedin/shiv>`_
+and quick-csr. 
+Shiv provides two helpful features:
+
++ create an interactive Python executable:
+  
+.. code-block:: console
+
+      $ shiv -o interactive-quick-csr.pyz .
+
+
+.. code-block:: console
+
+    $ interactive-quick-csr.pyz
+    Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 16:07:46) [MSC v.1900 32 bit (Intel)] on win32
+    Type "help", "copyright", "credits" or "license" for more information.
+    (InteractiveConsole)
+    >>> import quick_csr
+    >>>
+
++ create an executable that invokes our console script :code:`quick-csr` when executed:
+
+.. code-block:: console
+
+    $ shiv -o quick-csr-win7.pyz -c quick-csr .
+
+
+Setup without installing
```

Comment:
i don't get this section at all. probably it's better to have a section that tells people how to build a zip archive and another one that tells (rather different) people how to make use of these.

Reply:
okay, i adjusted my understanding by imagining another heading. `Using quick-csr packaged as a ZIP application` or similar nails the content.